services/node/node-current.tester.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+const { expect } = require('chai')
+const { Range } = require('semver')
+const t = (module.exports = require('../tester').createServiceTester())
+const {
+  mockPackageData,
+  mockCurrentSha,
+  mockNonExistingPackageData,
+} = require('./testUtils/test-utils')
+
+function expectSemverRange(message) {
+  expect(() => new Range(message)).not.to.throw()
+}
+
```

Comment:
Let's make sure we do still have live tests as well to provide that level of integration validation our service tests usually do. The mocked tests are great for this uncommon case where it's pretty tricky to test some of the internal logic within `node-version-color` with unit tests, but we definitely want to have the live tests as well.

Reply:
sure thing